### DISCUSSION TITLE
Allowing the `<code>` tag in the sidebar

### DISCUSSION THREAD

**Discussion by apcamargo** [mkdocs/mkdocs#3799] [Q&A] [answered]
When using backticks to create inline code blocks within headers, I noticed that the resulting `<code>` tags are not displayed in the sidebar.

<img width="1326" alt="image" src="https://github.com/user-attachments/assets/c12e881d-77d5-4f0b-b91c-5a309c2ff5bc">

Is there any way to allow the sidebar to show text within `<code>` tags?

**Comment by squidfunk** [comment]
Yes, the [typeset plugin](https://squidfunk.github.io/mkdocs-material/plugins/typeset/) is what you're looking for. Note that it's currently only available in [Insiders](https://squidfunk.github.io/mkdocs-material/insiders/).

**Reply by squidfunk** [reply, marked as answer]
Oops, I confused browser tabs and thought this question was asked on Material for MkDocs' discussion board 😅

So, to add to that, the typeset plugin only works with [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/), but which you appear to be using anyway. The maintainers of MkDocs have been asked several times for this feature, but until now, it hasn't been implemented yet. This is why Material for MkDocs rolled its own (temporary) solution.

**Reply by lovelydinosaur** [reply]
Thanks. Ties in neatly with some parts of our latest team call.
Could you interlink whichever tickets you're referencing?

**Reply by squidfunk** [reply]
For starters:

- #3532
- #3357

**Reply by apcamargo** [reply]
> Oops, I confused browser tabs and thought this question was asked on Material for MkDocs' discussion board 😅

Well, this makes two of us because I intended to open this discussion there. Thank you for the answer!
